tests/test_setup_logging.py

```diff
@@ -47,4 +47,5 @@ def tearDown(self):
         self.handler.close()
 
     def test_datadog_handler_called(self):
+        self.logger.warning("Testing handler 1 2 3")
         self.assertEqual(True, self.handler.send.called)
```

Comment:
I think a more thorough test is in order. Perhaps checking that the arguments to send are what you expect?

Reply:
The arguments here are the default log records as we are not modifying the content in any way. It's a `dict` that is generated by the logging module.

Reply:
I'll check for that message is exactly what's used.